### PR TITLE
fix: openssl boefje stuck on port 80

### PR DESCRIPTION
**Pull request by tobiasBDO**
### Changes
Added an if-statement in order drop non-https schemes from being picked up by openssl boefje.

### Issue link
OpenSSL boefje suffers from a similar fate as https://github.com/minvws/nl-kat-coordination/issues/1913, this change stops boefje from getting stuck during the processing of non-ssl OOI's in some cases, with the following output:
```sh
my-openkathost:/nl-kat-coordination$ openssl s_client -host [redacted-ip] -port 80 -prexit -showcerts -servername [redacted-host]

CONNECTED(00000003)
```
### Demo
After testing it indeed seems all the http-based schemes are passed automatically due to the added if-statement.

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: boefjes/boefjes/plugins/kat_ssl_certificates/main.py [hunk 1]
```diff
@@ -9,9 +9,13 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
     client = docker.from_env()
     input_ = boefje_meta.arguments["input"]
     hostname = input_["hostname"]["name"]
+    scheme = input_["ip_service"]["service"]["name"]
     ip_address = input_["ip_service"]["ip_port"]["address"]["address"]
     port = input_["ip_service"]["ip_port"]["port"]
 
+    if scheme != "https":
+        return [(set(), "")]
```
Comment (underdarknl) [inline boefjes/boefjes/plugins/kat_ssl_certificates/main.py:17]: I think we'd want to check for http (and possibly other services), and only then not scan as there are other services that we would like to check.
Returning an empty result is less usable than what we are currently doing for 'empty results':

`return [(set("info/boefje"), "Skipping check due to unsupported scheme.")]`

Comment (Donnype) [reply boefjes/boefjes/plugins/kat_ssl_certificates/main.py:17]: Note that we should be careful with this `set` notation!
```shell
$ python -c 'print(set("info/boefje"))'
{'e', 'f', 'n', 'o', 'i', 'b', 'j', '/'}
```

Comment (dekkers) [reply boefjes/boefjes/plugins/kat_ssl_certificates/main.py:17]:
```suggestion
    if scheme == "http":
        return [({"info/boefje"}, "Skipping check due to non-TLS scheme")]
```

**Review by ammar92 — APPROVED** [review]

**Comment by stephanie0x00** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
Can confirm that previous the boefje output would throw an error and that this fix nicely shows the 'Skipping' message. Looks good. 

### What doesn't work:
n/a

### Bug or feature?:
n/a

File: boefjes/boefjes/plugins/kat_ssl_certificates/main.py (hunk 1 above)
Comment (underdarknl) [reply boefjes/boefjes/plugins/kat_ssl_certificates/main.py]: Could you also flip the scheme check around?

**Review by underdarknl — APPROVED** [review]

**Pull request merged**